pkgs/tools/security/knockknock/default.nix

```diff
@@ -26,9 +25,8 @@ in python2Packages.buildPythonApplication rec {
   meta = with lib; {
     description = "Simple, secure port knocking daemon and client written in Python";
     homepage    = "http://www.thoughtcrime.org/software/knockknock/";
-    license     = licenses.gpl3;
+    license     = licenses.gpl3Plus;
     maintainers = with maintainers; [ copumpkin ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ copumpkin ];
    platforms   = platforms.unix;
```

Reply:
That's default for `buildPythonApplication`: https://github.com/NixOS/nixpkgs/blob/a1b8eb4a30e32553f47ddfb185816f587ba08bbc/pkgs/development/interpreters/python/mk-python-derivation.nix#L177 https://github.com/NixOS/nixpkgs/blob/92c881b6a72abce5bb2f5db3f903b4871d13aaa9/pkgs/development/interpreters/python/cpython/default.nix#L505

Reply:
I would still prefer to it be explicit, but ok.

Reply:
BTW

```
nix-repl> legacyPackages.x86_64-linux.knockknock.meta.platforms
[ "i686-cygwin" "x86_64-cygwin" "x86_64-darwin" "i686-darwin" "aarch64-darwin" "armv7a-darwin" "i686-freebsd" "x86_64-freebsd" "aarch64-genode" "i686-genode" "x86_64-genode" "x86_64-solaris" "js-ghcjs" "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux" "armv7l-linux" "i686-linux" "m68k-linux" "mipsel-linux" "powerpc64-linux" "powerpc64le-linux" "riscv32-linux" "riscv64-linux" "s390-linux" "s390x-linux" "x86_64-linux" "mmix-mmixware" "aarch64-netbsd" "armv6l-netbsd" "armv7a-netbsd" "armv7l-netbsd" "i686-netbsd" "m68k-netbsd" "mipsel-netbsd" "powerpc-netbsd" "riscv32-netbsd" "riscv64-netbsd" "x86_64-netbsd" "aarch64-none" "arm-none" "armv6l-none" "avr-none" "i686-none" "msp430-none" "or1k-none" "m68k-none" "powerpc-none" "riscv32-none" "riscv64-none" "s390-none" "s390x-none" "vc4-none" "x86_64-none" "i686-openbsd" "x86_64-openbsd" "x86_64-redox" "wasm64-wasi" "wasm32-wasi" "x86_64-windows" "i686-windows" ]
```

This is taking way more than just `linux` and `darwin`.

Reply:
OK, added.